src/strongbox/logging.clj

```diff
@@ -60,7 +61,7 @@
    :appenders {:println {:enabled? true
                          :async? false
                          :output-fn :inherit
-                         :fn anon-println-appender}}})
+                         :fn (anon-println-appender -colour-log-map)}}})
```

Comment:
making this an 'appender config' or some such, with 'colour-log-map' a key of it

Reply:
test no coloured output!